src/Analysis/GWASResults/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './DismissibleMessage.css';
+
+export const isEnterOrSpace = (event) => event.key === 'Enter'
+  || event.key === ' '
+  || event.key === 'Spacebar'
+  || event.keycode === '32'
+  || event.keycode === '13';
+
+const DismissibleMessage = ({
+  title = 'Placeholder Title',
+  description = 'placeholder description',
+  messageType = 'success',
+}) => {
```

Comment:
I think it would be good to try to reuse the one from GWASApp. Let's discuss!

Reply:
as agreed, duplication will be addressed in https://ctds-planx.atlassian.net/browse/VADC-509